scripts/generate-ebpf-docker.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
I don't see this script actually used anywhere? Could we update to generate-ebpf-types to build using Docker if `uname -s` is anything other than Linux?

Reply:
or rather have a `generate-ebpf-docker.sh` and `generate-ebpf-native.sh` and a wrapper shell program `generate-ebpf-types.sh` that calls either script depending on platform.